src/utils/logging/logDebugTexture.ts

```diff
@@ -1,6 +1,14 @@
 import type { Texture } from '../../rendering/renderers/shared/texture/Texture';
 import type { Renderer } from '../../rendering/renderers/types';
 
+/**
+ * Logs a texture to the console as a base64 image.
+ * This can be very useful for debugging issues with rendering.
+ * @param texture - The texture to log
+ * @param renderer - The renderer to use
+ * @param size - The size of the texture to log in the console
+ * @memberof utils
```

Comment:
we can ignore this function as it actually is used when I need to log a texture mid render (which no one should be doing). 

Users can use `renderer.extract.log(thing)` for most use cases